config/config.go

```diff
@@ -39,12 +39,6 @@ type Config struct {
 
 	ContainerRuntimeExecutors ContainerRuntimeExecutors `json:"containerRuntimeExecutors,omitempty"`
 
-	// KubeletPort is needed when using the kubelet containerRuntimeExecutor, default to 10250
-	KubeletPort int `json:"kubeletPort,omitempty"`
```

Comment:
Could you also update the example configmap in docs folder to add a comment saying that these will be deprecated?

Reply:
done